etc/httpd/conf.d/ssl.conf

```diff
@@ -17,7 +17,7 @@ LogLevel warn
 
 SSLEngine on
 
-SSLProtocol all -SSLv2
+SSLProtocol all -SSLv2 -SSLv3 -TLSv1 -TLSv1.1
```

Comment:
The examples don't mention `-SSLv2`, only `-SSLv3`. Is it still necessary to remove `SSLv2`?